src/components/routes/Booking/BookingQuote/BookingQuote.tsx

```diff
@@ -51,7 +51,7 @@ const BookingQuote = ({ booking, currency, fromBee }: Props) => {
             {host.firstName}
```

Comment:
how about we show:
## Listing Title
[small] homeType
[small] City

Reply:
listing titles can get very long

![image](https://user-images.githubusercontent.com/18321302/51711598-5e32c900-1fe1-11e9-9653-a70233376451.png)

Reply:
we can truncate the listing title to a set number of letters if we want the title

Reply:
Can we try this:
Replace Trip Details with Listing title and go one size smaller.
Replace "31 Nights at Listing Title" with "homeType in Los Angeles"

Reply:
<img width="413" alt="screen shot 2019-01-24 at 6 13 03 pm" src="https://user-images.githubusercontent.com/7976/51720756-bb3f7680-2003-11e9-8ced-9399d072995a.png">

Reply:
<img width="493" alt="screen shot 2019-01-24 at 6 19 50 pm" src="https://user-images.githubusercontent.com/7976/51721004-afa07f80-2004-11e9-9b30-feaff29fdf2f.png">

Reply:
<img width="430" alt="screen shot 2019-01-24 at 6 21 04 pm" src="https://user-images.githubusercontent.com/7976/51721033-e70f2c00-2004-11e9-9721-f06c7a285cb8.png">

Reply:
i like the idea of replacing the "31 nights" thing.  i think we should still truncate just in case since we dont limit the amount of characters for the listing title for non-beenest providers

Reply:
attempt at sketch edit :)
https://projects.plantapp.io/p/18560/13/DBB46102-0CA3-4E10-B89A-B13EB95A5C4F?r=13
<img width="295" alt="screen shot 2019-01-24 at 6 34 46 pm" src="https://user-images.githubusercontent.com/7976/51721461-c051f500-2006-11e9-8506-941afb385e5e.png">

Reply:
yea we can truncate text with css to two lines?

Reply:
doesn't seem like there's an easy way to truncate to 2 lines.
https://stackoverflow.com/questions/5269713/css-ellipsis-on-second-line

@kcvan how long do the titles get? I think 3 lines might be passable.

Reply:
@jerminatorhits if it's not uploaded from us, I'm not sure how long the titles can get, since we don't control the listing title length that come from other providers

Reply:
ahh i see. Long titles will be an issue across the app (Listing Cards, Booking Cards, mobile views, etc.) If we solve it here, we can do the same in other components!